README.md

```diff
@@ -13,24 +14,24 @@
 [gitter-image]: https://badges.gitter.im/tldr-pages/tldr.svg
 [prs-merged-url]: https://github.com/tldr-pages/tldr/pulls?q=is:pr+is:merged
 [prs-merged-image]: https://img.shields.io/github/issues-pr-closed-raw/tldr-pages/tldr.svg?label=merged+PRs
-[issuestats-url]: http://isitmaintained.com/project/tldr-pages/tldr
-[issuestats-image]: http://isitmaintained.com/badge/resolution/tldr-pages/tldr.svg
+[issuestats-url]: https://isitmaintained.com/project/tldr-pages/tldr
+[issuestats-image]: https://isitmaintained.com/badge/resolution/tldr-pages/tldr.svg
```

Comment:
I'm not sure that the time to issue resolution is really valuable here. Perhaps we should remove it, but that sounds like a separate issue to the one being discussed here.

Reply:
It's separate though. Let's do it in a different PR. And yes, I agree it's not valuable.

Reply:
For context, it was meant as a concrete signal to visitors that contributions are welcomed in this project, and regularly integrated, as a way to encourage people to get involved. That's also the purpose of the "number of contributors" and "merged PRs" badges.

We can keep that badge only in CONTRIBUTING.md, as we've done with the CLA badge for example. If we do so, would we want to move all three? I think they sort of work as a group.

Reply:
The problem is that the metrics used by isitmaintained are incorrect and not what we would use to judge "activity" in our org. They measure how fast issues are closed, and what percentage of issues are open. Both of which are skewed in our case, because we have issues which are several years old. Mostly because those are discussions which we never got around to implement. That has got nothing to do with merging regular PRs.

If there is some other website that shows the contribution graph, then I am happy to include that.

Reply:
That's a good point, but I'm not sure the first metric (the only one we use — how fast issues are closed) is skewed in the statistical sense, considering that they use the median rather than the mean, and that they ignore issues older than 6 months, as described [here](https://isitmaintained.com/project/tldr-pages/tldr).

However, it may be skewed in the sense that IsItMaintainer counts both issues and PRs, rater than just PRs, and we tend to be more forgiving with long-lived issues\* than most open source projects, adopting an [eventualist](https://meta.wikimedia.org/wiki/Eventualism) philosophy that doesn't strive for closing issues as fast as possible, but instead favors actual resolution of the original issue (even though some some of us are sometimes more impatient — I'm looking at you, @zdroid! 😜).

Indeed a PR resolution metric would be more useful for the message we want to convey here. They don't seem to support that, and I didn't find alternatives that do, so yeah, maybe we should just remove the badge.

<sub>\* sometimes PRs too, but that's much rarer and probably statistically insignificant.</sub>

Reply:
For you Golangers: I found two Go-based projects ([bitbadger](https://github.com/Pixep/bitbadger) and [nisekoi](https://github.com/UnsafePointer/nisekoi)) that seem to focus precisely on average PR merging time, but I suppose they'd need some massaging to feed an endpoint that would generate data we could pass to, say, a [dynamic shields.io badge](https://shields.io/#dynamic-badge). Do any of those look like something we could adapt without too much work?

Reply:
FYI, we are called "Gophers" :stuck_out_tongue:. I don't know, personally I don't find any value in badges like these. To me, if I want to see if a project is active - I just look at the last commit data, and if the issue tracker is active or not. Both of which are perfectly fine in our case. I am fine with average PR merging time, but wouldn't bother about spending too much effort on that.

Reply:
Ok, I'll remove the badge in an upcoming PR.

Reply:
Sounds good! Sorry for not getting back sooner with this. Let's fiddle with this badge in a future PR. For now I'll approve this PR.

Reply:
Update: I've opened #4103 to remove the badge.